appveyor.yml

```diff
@@ -120,20 +119,20 @@ test_script:
 
 # pushing all files in build directory recursively
 artifacts:
-- path: breeze_out\*.zip
+- path: Breeze.UI\app-builds\*.exe
   name: Breeze app
   
 #----------------------------------#
 #      deployment configuration    #
 #----------------------------------#
 
-#deploy:
-#- provider: GitHub
-#  tag: $(APPVEYOR_REPO_TAG_NAME)
-#  auth_token:
-#    secure: faXallYrtShJywqNK+adalNNl50Wjns7dFsCDDx5/eNBegshCLjfshUeMPgIX9Ld
-#  draft: false
-#  prerelease: true
-#  force_update: true
-#  on:
-#    branch: master                 # release from master branch only
+# deploy:
```

Comment:
Please uncomment this and I'll make the necessary change for it to be deployed properly.